content/en/docs/quickstart.md

```diff
@@ -143,9 +137,22 @@ Create a `trustpolicy.json` with the following trust policy in the notation conf
         }
     ]
 }
+EOF
+```
+
+Use `notation policy import` to import the trust policy configuration from a JSON file. For example:
+
+```shell
+notation policy import ./my_policy.json
+```
+
+Use `notation policy show` to view the applied policy configuration. For example:
+
+```shell
+notation policy show
 ```
 
-The above JSON creates a trust policy named `wabbit-networks-images`. The policy has `registryScopes` set to `*`, which applies the policy to all the artifacts of any registry. The `signatureVerification` is set to `strict`, which checks all validations and any failure will fail the signature verification. This policy uses the `wabbit-networks.io` trust store of type `ca` which was created in the previous step. For more details on trust policies, see [trust policy spec]({{< ref "/docs/concepts/trust-store-trust-policy-specification#trust-policy-properties" >}}).
+The above JSON creates a trust policy named `wabbit-networks-images`. The policy has `registryScopes` set to `*`, which applies the policy to all the artifacts of any registry. The `signatureVerification` is set to `strict`, which checks all validations and any failure will fail the signature verification. This policy uses the `wabbit-networks.io` trust store of type `ca` which was created in the previous step. For more details on trust policies, see [trust policy spec](https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-policy).
 
 To enable trust policy for specific repositories, set the `registryScopes` to those specific repositories. For example:
```

Comment:
Can you also fix the example below as it is not a valid json? It misses a `"` at the beginning.

Reply:
Good catch. Resolved.